src/io/gff.rs

```diff
@@ -222,58 +222,54 @@ impl<W: io::Write> Writer<W> {
 }
 
 /// A GFF record
-#[derive(Debug, Default, Serialize, Deserialize, Clone)]
+///
+#[derive(
+    Debug,
+    Default,
+    Serialize,
+    Deserialize,
+    Clone,
+    PartialEq,
+    Eq,
+    Getters,
+    Setters,
+    MutGetters,
+    Builder,
+)]
 pub struct Record {
+    #[getset(get, set, get_mut)]
+    #[builder(default)]
```

Comment:
instead of annotating every single field, you can annotate the struct itself with a default, e.g.:
```rust
#[get = "pub", get_mut = "pub", set = "pub"]  // generate public getters/setters for all fields in the struct
pub struct Record { ... }
```
I *think* the same goes for `builder`